### PR TITLE
run test without heavy test

### PR DESCRIPTION
**Pull request by zhfnjust**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "node": ">=12.0.0"
   },
   "scripts": {
-    "test": "mocha -r ts-node/register 'tests/**/*.scrypttest.*'  --reporter spec --timeout 600000 -g '' ",
+    "test": "npm run test:withoutHeavy",
+    "test:withoutHeavy": "mocha -r ts-node/register 'tests/**/*.scrypttest.*'  --reporter spec --timeout 10000 -i -g 'Heavy'",
```
Comment (xhliu) [inline package.json:11]: 👍 nice

-f may be better than -g:
See Running matching tests in https://blog.logrocket.com/a-quick-and-complete-guide-to-mocha-testing-d0e0ea09f09d/

File: .travis.yml [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
-      $HOME/.cargo/bin/cargo build --release
-      $HOME/.cargo/bin/cargo install --path circom
-      cd ..
-      if [ $TRAVIS_OS_NAME = "windows" ]; then
-        export PATH=$PATH:$HOME/.cargo/bin
-      else
-        source $HOME/.cargo/env
-      fi
-    fi
+  # - |
```
Comment (xhliu) [inline .travis.yml:7]: Uncomment these when merging?

File: .travis.yml [hunk 3]
```diff
@@ -1,38 +1,48 @@
...
-  - npx scryptlib download
-  - |
-    if [ $TRAVIS_OS_NAME = "osx" ]; then
-      npm run download:circom
-    fi
-  - npm t
\ No newline at end of file
+jobs:
+  include:
+    - if: branch = master AND type = push
```
Comment (xhliu) [inline .travis.yml:25]: Will this run in a PR not yet merged to `master`?

**Review by xhliu — APPROVED** [review]
💯 good trick on Mocha.

Let's merge after #193 , which conflicts w/ this.

File: .travis.yml (hunk 3 above)
Comment (zhfnjust) [reply .travis.yml]: yes

**Pull request merged**